index/index.ml

```diff
@@ -55,24 +74,50 @@ let main
       close_in h ;
       files @ other_files
   in
+  let files = List.map Fpath.v files in
+  let favourite_files = List.map Fpath.v favourite_files in
```

Comment:
Should we be using `Fpath.of_string`? (as [the doc mentions it's better for untrusted inputs](https://ocaml.org/p/fpath/0.7.3/doc/Fpath/index.html#val-v))

Reply:
It seems to me that `of_string` return the same error but in a result instead of an exception. Here, if we got an invalid path I think we would crash, so I think `v` is fine.
We might want to switch to `of_string` if we wanted to have a warning instead of a crash of invalid paths.